pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -2168,6 +2190,18 @@ type NetworkKubeSpan struct {
 	KubeSpanAllowDownPeerBypass bool `yaml:"allowDownPeerBypass,omitempty"`
 }
 
+// NetworkDeviceSelector struct describes network device selector.
+type NetworkDeviceSelector struct {
+	// description: PCI, USB bus prefix.
```

Comment:
nit: I would add a small note on wildcard matching for each field, just for the documentation purposes